Traceback (most recent call last):
  File "/work/nikki/MT/code/service.py", line 41, in <module>
    read_file()
  File "/work/nikki/MT/code/service.py", line 30, in read_file
    df = pd.read_excel(file_path)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work/nikki/miniconda/lib/python3.12/site-packages/pandas/io/excel/_base.py", line 495, in read_excel
    io = ExcelFile(
         ^^^^^^^^^^
  File "/work/nikki/miniconda/lib/python3.12/site-packages/pandas/io/excel/_base.py", line 1550, in __init__
    ext = inspect_excel_format(
          ^^^^^^^^^^^^^^^^^^^^^
  File "/work/nikki/miniconda/lib/python3.12/site-packages/pandas/io/excel/_base.py", line 1402, in inspect_excel_format
    with get_handle(
         ^^^^^^^^^^^
  File "/work/nikki/miniconda/lib/python3.12/site-packages/pandas/io/common.py", line 882, in get_handle
    handle = open(handle, ioargs.mode)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: './data/Question Type examples 9_20_24'
srun: error: g001: task 0: Exited with exit code 1
